test/active_record_doctor/detectors/missing_unique_indexes_test.rb

```diff
@@ -302,6 +302,45 @@ def test_case_insensitive_compound_unique_lower_index_exists
     refute_problems
   end
 
+  def test_case_insensitive_unique_index_with_citext
+    skip unless postgresql?
+
+    ActiveRecord::Base.connection.execute(<<-SQL)
```

Comment:
Are we able to use `enable_extension` here?

Reply:
Done :)